aws/basic-eks-cluster/xebialabs/xlr-pipeline-ci-cd.yaml.tmpl

```diff
@@ -16,6 +15,36 @@ spec:
     - {{$app}}
     scriptUsername: !value XL_RELEASE_USERNAME
     scriptUserPassword: !value XL_RELEASE_PASSWORD
+
+    variables:     
+      - key: AWSaccessKey
```

Comment:
This seems unnecessary

Reply:
not if the user did not use .aws/credentials

Reply:
These are not used anywhere so what is the purpose of this here?